.github/workflows/integration-tests.yaml

```
@@ -0,0 +1,67 @@
+name: integration-tests
+on:
+  push:
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ '8' ]
```

We should be testing on `11` as well.

For example, all XML parsing utilities are gone since JDK 9, I wonder if this is not problematic...

```suggestion
        java: [ '8', '11' ]
```